$NetBSD: patch-CMakeLists.txt,v 1.1 2025/08/15 23:12:31 gdt Exp $

Kludge around cmake requirement that the minimum version be high
enough.
	
--- CMakeLists.txt.orig	2025-08-15 23:08:20.897300798 +0000
+++ CMakeLists.txt
@@ -31,7 +31,7 @@
 #
 #  Please report all bugs and problems to <gl2ps@geuz.org>.
 
-cmake_minimum_required(VERSION 2.8 FATAL_ERROR)
+cmake_minimum_required(VERSION 3.22 FATAL_ERROR)
 
 # do not warn about non-definition of WIN32 on Cygwin
 set(CMAKE_LEGACY_CYGWIN_WIN32 0)
